agenta-web/src/components/Playground/ViewNavigation.tsx

```diff
@@ -25,21 +25,10 @@ const ViewNavigation: React.FC<Props> = ({
 }) => {
     const router = useRouter()
     const appName = router.query.app_name as unknown as string
-    const {
-        inputParams,
-        optParams,
-        URIPath,
-        isLoading,
-        isError,
-        error,
-        isParamSaveLoading,
-        saveOptParams,
-    } = useVariant(appName, variant)
+    const {inputParams, optParams, URIPath, isError, error, isParamSaveLoading, saveOptParams} =
+        useVariant(appName, variant)
     const [isParamsCollapsed, setIsParamsCollapsed] = useState("1")
 
-    if (isLoading) {
```

Comment:
Instead of removing `isLoading` altogether, please use the antd Spin component to show the loader while retain the state at the same time.

Like this:
<img width="759" alt="image" src="https://github.com/Agenta-AI/agenta/assets/17180132/628edf1a-2f12-4e1e-acf1-a37d73735b02">

Reply:
Thanks for the review @MohammedMaaz